demo/molecule/thumbnail/demo/index.scss

```diff
@@ -0,0 +1,19 @@
+@import '~@s-ui/react-atom-spinner/lib/index';
+
+.DemoMoleculeThumbnail {
+  padding: 20px;
+
+  &-table {
+    margin-bottom: 20px;
+  }
+
+  &-cell {
+    width: 25%;
+    vertical-align: top;
+    text-align: center;
+  }
+
+  .sui-AtomSpinner-loader {
```

Comment:
This breaks AtomSpinner encapsulation, you shouldn't override another component's styles, even inside a demo because someone will end up following this bad practice

Reply:
Removed, but there's an issue w/ `AtomSpinner` that makes it not to be exactly centered when used inside of a container. Maybe we should create a task to fix this.

Reply:
Yes, could you please create this task?